AkshayaP/00 - Projects/04-Frontend Projects/Employee Management System/Assets/JavaScript/Admin/Employee_details.js

```diff
@@ -0,0 +1,163 @@
+let EmployeeDetailMethods = {};
+
+const DIALOG_EMPLOYEE_REGISTER = document.getElementById("dialog-employee-register");
+const BTN_ADD_EMPLOYEE = document.getElementById("add-employee");
+
+const INP_EMPLOYEE_NAME = document.getElementById('employee-name');
+const INP_EMPLOYEE_AGE = document.getElementById('employee-age');
+const INP_EMPLOYEE_ADDRESS = document.getElementById('employee-address');
+const INP_EMPLOYEE_QUALIFICATION = document.getElementById('employee-qualification');
+const INP_EMPLOYEE_POSITION = document.getElementById('employee-position');
+const INP_EMPLOYEE_SALARY = document.getElementById('employee-salary');
+const INP_EMPLOYEE_GENDER_MALE = document.getElementById('genderMale');
+const INP_EMPLOYEE_GENDER_FEMALE = document.getElementById('genderFemale');
+//const INP_EMPLOYEE_PHOTO = document.getElementById('EmployeePhoto');
+
+document.addEventListener("DOMContentLoaded", function()
+{
+    EmployeeDetailMethods.Initialize();
+});
+
+EmployeeDetailMethods.Initialize = function()
+{
+    BTN_ADD_EMPLOYEE.addEventListener("click", EmployeeDetailMethods.OpenEmployeeRegisterDialog);
+    EmployeeDetailMethods.Display();
+}
+
+EmployeeDetailMethods.OpenEmployeeRegisterDialog = function()
+{
+    DIALOG_EMPLOYEE_REGISTER.showModal();
+}
+
+EmployeeDetailMethods.Save = function()
+{
+    const NAME = INP_EMPLOYEE_NAME.value;
+    const AGE = INP_EMPLOYEE_AGE.value;
+    const ADDRESS = INP_EMPLOYEE_ADDRESS.value;
+    const QUALIFICATION = INP_EMPLOYEE_QUALIFICATION.value;
+    const POSITION = INP_EMPLOYEE_POSITION.value;
+    const SALARY = INP_EMPLOYEE_SALARY.value;
+    const GENDER = INP_EMPLOYEE_GENDER_MALE.checked ? 'Male' : INP_EMPLOYEE_GENDER_FEMALE.checked ? 'Female' : '';
```

Comment:
const GENDER = INP_EMPLOYEE_GENDER_MALE.checked ? 'Male' : 'Female';

Can we do like this? It's either male or female right?

Reply:
done